riscv/decode.h

```diff
@@ -2048,6 +2072,8 @@ reg_t index[P.VU.vlmax]; \
   VI_VFP_COMMON \
   require((P.VU.vsew == e16 && p->extension_enabled('F')) || \
           (P.VU.vsew == e32 && p->extension_enabled('D'))); \
```

Comment:
Shouldn't these lines be deleted?

Reply:
Yes. This may be added when rebased to upstream. I'll delete it later.